Common/src/main/java/at/petrak/hexcasting/xplat/IXplatAbstractions.java

```diff
@@ -124,6 +125,11 @@ public interface IXplatAbstractions {
     @Nullable
     ADIotaHolder findDataHolder(Entity entity);
 
+    @Nullable
+    default ADIotaHolder findDataHolder(BlockPos pos, ServerLevel world){
+        return HexAPIImpl.getBlockyIotaHolder(world, pos);
+    }
+
```

Comment:
It feels kinda weird to put something in IXplatAbstractions if it's not actually different per-platform. I see why it's there, I guess... maybe add a comment justifying it?